README.md

```diff
@@ -1,48 +1,172 @@
+        **Note:** Make sure to save the password for future use.
+
+        * Option 2: Partner who already used the Collector\
+        Under the `partnerName` entry enter your partner name,
+        and under the `collectorAppPassword` entry enter the password
+        you defined in your first use of the collector.
+
+    Example of filled entries in CNF Certifiaction configuration file,
+    to allow data collection by partner name:
+
+    ```sh
+    executedBy: "Partner"
+    partnerName: "partner_example"
+    collectorAppPassword: "password_example"
+    ```
+
+3. Run CNF Certifiaction suites with the adjusted configuration file.\
+ (see [CNF Certification Test description](https://test-network-function.github.io/cnf-certification-test/test-container/))
 
-`./scripts/send-to-collector.sh "enter_endpoint" "path/to/claim.json" "enter_executed_by" "enter_partner_name(optional)" "enter_password(optional)"`
+#### Option 2 - send a claim.json file directly to Collector
 
-# Instructions for running get-from-collector.sh
+If you haven't already, clone Collector's repository:
+
+```sh
+git pull https://github.com/test-network-function/collector.git
+```
 
 From collector's repo root directory, use the following command:
 
-`./scripts/get-from-collector.sh "enter_endpoint" "enter_partner_name" "enter_password"`
+<!-- markdownlint-disable -->
+```sh
+./scripts/send-to-collector.sh "enter_endpoint" "path/to/claim.json" "enter_executed_by" "enter_partner_name(optional)" "enter_password(optional)"
+```
+<!-- markdownlint-enable -->
+
+**Note:** To get Collector's endpoint, contant us by email (mentioned at the bottom).
+
+### Get data from Collector
+
+#### Option 1 - For both Admin and Partners
+
+Partners who use collector to store data by their name,
+can have acess to their saved data.\
+If you haven't already, clone Collector's repository:
 
-# Instructions for running sanity-check.sh
+```sh
+git pull https://github.com/test-network-function/collector.git
+```
 
 From collector's repo root directory, use the following command:
 
-`ENDPOINT=enter_endpoint COLLECTOR_USERNAME=enter_partner_name COLLECTOR_PASSWORD=enter_password ./scripts/sanity-check.sh`
+```sh
+./scripts/get-from-collector.sh "enter_endpoint" "enter_partner_name" "enter_password"
+```
+
+**Note:** To get Collector's endpoint, contant us by email
+(mentioned at the bottom).
+
+#### Option 2 - For Admin only
+
+Access the data through
+[Collector's grafana dashboard](http://44.195.143.94:3000/d/e5530a23-24b9-4e7f-ab28-8e778d99f429/collector-s-dashboard?orgId=1).
+
+## Run Collector Locally
+
+### Prerequisites
+
+* Docker or Podman
+* MySQL
+
+### Build and Run Collector's container locally
+
+Use the Following commands to build and run Collector's container and database locally:
+
+* **Clone Collector's repository:**
+
+    ```sh
+    git pull https://github.com/test-network-function/collector.git
+    ```
+
+* **(Optinal) Build and Push your Collector image:**\
```

Comment:
This renders to `(Optinal) Build and Push your Collector image:\` with slash at the end.  What is a purpose of slashes in other places?

Reply:
It makes the next line to be in a seperate line and not continuing the current (similar to \n). If you'll look at the preview of the README you'll see it does not appear.

Reply:
Oh right, thanks!